### ISSUE TITLE
Status and Payment status are not translated in show feature and recordPayment feature

### ISSUE THREAD

**Issue by Gregsai** [idurar/idurar-erp-crm#881]
**Describe the bug**

When you go to the invoice list/quote/offer, select an item and choose the "show" option, the status information is not translated in the user language. For example, my language is french, and when i display an invoice item, here is what is displayed : 

![image](https://github.com/idurar/idurar-erp-crm/assets/85080566/25bfd53b-8a82-45db-b1f7-1cbd961d369d)

Instead of unpaid, it should display "impayé". But the bug is the same for all languages and for quote and offer as well. 

**To Reproduce**

1. Go to "Invoice or quote or offer"
2. Choose one of the items
3. Click on the 3 dots
4. Choose "Show"
5. Change your language to anything but english.
6. See error (status is displayed in english instead of the user's language)

**Expected behavior**

status is displayed in the user language. 

**Desktop (please complete the following information):**
- OS: Windows 11
- Browser [Brave; Edge]
- Version [1.60.125; 119.0.2151.93]

**Comment by Gregsai** [comment]
I submitted a PR to solve this issue. 

This is the PR #882

**Comment by Gregsai** [comment]
I found a similar bug : 

1. Go to invoice
2. Select an item
3. Click on the 3 dots
4. Choose Record Payment
5. Change the language into one different than english
6. See that the status of the invoice as well as the payment status is still in english

![image](https://github.com/idurar/idurar-erp-crm/assets/85080566/aa22611c-bdf0-4a85-8165-64ddd8c0dfe5)

I don't think that i should open a new issue so i am raising it here. Sorry I did not noticed it before the first PR.
